doc/source/themes/nature_with_gtoc/layout.html

```diff
@@ -19,7 +19,7 @@
                 {%- block sidebar1 %}
                 {%- block sidebartoc %}
                 <h3>{{ _('Table Of Contents') }}</h3>
-                {{ toctree() }}
+                {{ toctree(includehidden=True) }}
```

Comment:
We use a few hidden toctrees in the API docs to prevent sphinx /numpydoc from throwing a warning (e.g. reference/index.rst has `api/pandas.api.extensions.ExtensionDtype.na_value`. Do you know if those show up in the sidebar now? Or is the `include_hidden` limited to just certain pages?

Reply:
Good question. And yes, they do ... 
Checking now if by putting them in a comment instead of using `:hidden:` helps for this.

Reply:
Another option may be to put them in a different toctree on a page that is an orphan? Not sure if Sphinx will complain about that.

Reply:
Both seemed to work, used the commented toctrees now.